src/database/migrations/2018_03_31_104532_create_media_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMediaTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('media', function (Blueprint $table) {
+            $table->increments('id');
+            $table->timestamps();
+            $table->softDeletes();
+            $table->string('link');
+            $table->string('name')->nullable();
+            $table->boolean('is_public')->default(false);
+            $table->integer('owner_id')->nullable();
+
+            if (config('database.default') == 'mysql') {
+                $table->jsonb('meta')->nullable();
+            } else {
+                $table->jsonb('meta')->default('{}');
+            }
+
+            $table
+                ->foreign('owner_id')
+                ->references('id')
+                ->on('users')
```

Comment:
please get table name from the configurable User model

Reply:
Done.